assets/app/view/game/part/divisions.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+require 'lib/color'
```

Comment:
what is this? is this new code? i don't think i've reviewed this yet. 

if it's new, i can't really review this if it's a moving target

Reply:
I'm so sorry. This is part of the code I'm working on now and for some reason it escaped `git stash`

Reply:
I'll stop using "amend" and force push. I don't seem to find a satisfactory way to send the revisions.